app/models/organisation.rb

```diff
@@ -492,6 +500,7 @@ def organisations_with_services_and_information_link
       driver-and-vehicle-standards-agency
       environment-agency
       high-speed-two-limited
+      highways-england
```

Comment:
It will, so we'll have to delete the extra link for that page on this is deployed